requirements.txt

```diff
@@ -1,4 +1,5 @@
 annoy
+ga4gh.vrs[extras]
```

Comment:
I think the version here should probably be pinned so when we release a vrs-python non-alpha/beta on a 1.x or 2.x tag this doesn't break. The version discussed previously was `0.8.4` but assuming vrs-python sticks by semantic versioning it could be something wider.